### DISCUSSION TITLE
Ansible Examples for Teleport Configuration

### DISCUSSION THREAD

**Discussion by jon-can** [gravitational/teleport#5579] [Q&A]
Are there any Ansible examples that demonstrate configuring Teleport?

**Comment by klizhentas** [comment]
We do, but it was never published. Here is a full ansible + etcd setup: 
https://github.com/gravitational/ops/tree/fspmarshall/monster/loadtest

**Reply by leftyfb** [reply]
The link doesn't work

**Reply by webvictim** [reply]
Ah yes, it's a private repo. We have an issue to tidy up and publish it: #5601
